packages/core/src/controllers/NetworkController.ts

```diff
@@ -68,6 +68,37 @@ export const NetworkController = {
     state.requestedCaipNetworks = requestedNetworks
   },
 
+  getRequestedCaipNetworks() {
```

Comment:
You can use `sortRequestedNetworks` in `CoreHelperUtil.ts` to return sorted networks.